### PR TITLE
added activity labels

### LINKED ISSUE #1050: Add Activities Label
image

- Need something to label activities and socials

### LINKED ISSUE #1051: Separate Calendar Tags
image

- Some Tags are meant for HACKATHON calendar while others are meant for HACKATHON_LEADS calendar
- Make the separation so its easier to see which are hackathon events and which are leads events
- It is simply a visual separation of labels

### PR DESCRIPTION
**Pull request by JohnZ9865**
closes #1050 
closes #1051 
image

### CODE REVIEW

**Comment by JohnZ9865** [comment]
image

**Comment by JohnZ9865** [comment]
Also, there's this error here which I saw after "git reset --hard origin/dev": 
image
I simply ignored it but it exists.

File: src/components/dynamic/admin/services/calendar/Toolbar.jsx [hunk 1]
```diff
@@ -87,16 +87,41 @@ const CustomToolbar = ({
...
-            classes="my-1"
-            onClick={() => onClick(label)}
-          />
-        ))}
+
+      <div className="flex flex-col">
+        <div className="flex justify-evenly items-center flex-wrap">
+          <p>Leads Events:</p>
+          {Object.entries(LABELS)
+            .filter(function (obj) {
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Toolbar.jsx]: arrow functions

File: src/components/dynamic/admin/services/calendar/Toolbar.jsx [hunk 2]
```diff
@@ -87,16 +87,41 @@ const CustomToolbar = ({
...
+                classes="my-1"
+                onClick={() => onClick(label)}
+              />
+            ))}
+        </div>
+
+        <div className="flex justify-evenly items-center flex-wrap">
+          <p>Hackathon: </p>
+          {Object.entries(LABELS)
+            .filter(function (obj) {
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Toolbar.jsx]: arrow function

File: src/components/dynamic/admin/services/calendar/Toolbar.jsx [hunk 3]
```diff
@@ -87,16 +87,41 @@ const CustomToolbar = ({
...
-            onClick={() => onClick(label)}
-          />
-        ))}
+
+      <div className="flex flex-col">
+        <div className="flex justify-evenly items-center flex-wrap">
+          <p>Leads Events:</p>
+          {Object.entries(LABELS)
+            .filter((obj) => {
+              return obj[1].eventType == "leads";
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Toolbar.jsx]: implicit return, what is the structure of obj?

File: src/components/dynamic/admin/services/calendar/Toolbar.jsx [hunk 4]
```diff
@@ -87,16 +87,41 @@ const CustomToolbar = ({
...
+                onClick={() => onClick(label)}
+              />
+            ))}
+        </div>
+
+        <div className="flex justify-evenly items-center flex-wrap">
+          <p>Hackathon: </p>
+          {Object.entries(LABELS)
+            .filter((obj) => {
+              return obj[1].eventType != "leads";
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Toolbar.jsx]: implicit return and !==

Comment (JohnZ9865) [reply src/components/dynamic/admin/services/calendar/Toolbar.jsx]: I fixed them

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**